app/controllers/static_pages_controller.rb

```diff
@@ -1,4 +1,6 @@
 # Just a controller to set up an initial root page to test Heroku deployment
 class StaticPagesController < ApplicationController
-  def index; end
+  def index 
+    @games = Game.all
```

Comment:
Another idea, since we don't need to load all the games. We can load the available games only.

`@available_games = Game.available`

Then in your view, could do `@available_games.each`

Reduces the number of queries this code is doing